openslides/core/static/js/core/base.js

```diff
@@ -1438,6 +1438,26 @@ angular.module('OpenSlidesApp.core', [
     };
 })
 
+// Wraps the orderBy filter. But puts ("", null, undefined) last.
+.filter('orderByEmptyLast', [
+    '$filter',
+    function ($filter) {
+        return function (array, sortPredicate, reverseOrder, compareFn) {
+            var falsyItems = [];
+            var truthyItems = _.filter(array, function (item) {
+                var falsy = item[sortPredicate] === undefined ||
```

Comment:
Please use `void 0` instead of `undefined`.

Reply:
Done.